src/main/java/de/tum/in/www1/artemis/service/messaging/DistributedInstanceMessageSendService.java

```diff
@@ -162,10 +168,6 @@ public void sendParticipantScoreSchedule(Long exerciseId, Long participantId, Lo
         sendMessageDelayed(MessageTopic.PARTICIPANT_SCORE_SCHEDULE, exerciseId, participantId, resultId);
     }
 
-    private void sendMessageDelayed(MessageTopic topic, Long payload) {
-        exec.schedule(() -> hazelcastInstance.getTopic(topic.toString()).publish(payload), 1, TimeUnit.SECONDS);
-    }
-
```

Comment:
Got an editor warning, so I've removed that as it's practically doing the same as the method below, right?

Reply:
Ok apparantly not acc. to @krusche, will revert :)

Reply:
Remember to revert this change, the difference in signature is small but important.

Reply:
Sure, done!